lib/parsers/parsers.ts

```diff
@@ -61,7 +50,7 @@ export function parseOutput(gptString: string): ParsedOutput {
       .trim();
     completed =
       completedString.startsWith("true") ||
-      (completedString.startsWith("question") ? null : false);
+      completedString.startsWith("question");
```

Comment:
This is a tentative fix for the issue where superflows gets stuck in an endless loop

Reply:
Fixed properly